[quant][docs] quantized model save/load instructions

supriyar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #69789

Summary:
Add details on how to save and load quantized models without hitting errors
Address issue #69426
Test Plan:
cd docs
make html

preview docs page, looks good.

![image](https://user-images.githubusercontent.com/2657489/145655866-f1bf7917-6861-412b-9703-676dff63263d.png)


Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33030991](https://our.internmc.facebook.com/intern/diff/D33030991)

supriyar committed:
[quant][docs] quantized model save/load instructions
Summary:
Add details on how to save and load quantized models without hitting errors

Test Plan:
CI autogenerated docs

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[quant][docs] quantized model save/load instructions"


Summary:
Add details on how to save and load quantized models without hitting errors

Test Plan:
CI autogenerated docs

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar commented:
@supriyar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33030991).

supriyar commented:
@pytorchbot ciflow rerun -l ciflow/docs

jerryzh168 commented on docs/source/quantization.rst:
```diff
@@ -816,6 +816,55 @@ An e2e example::
+1. Saving/Loading the quantized model state_dict
+2. Saving/Loading scripted quantized models using ``torch.jit.save`` and ``torch.jit.load``
+
+An example::
```
nit: to make it clearer, can we one two examples, one for each case

supriyar committed:
Update on "[quant][docs] quantized model save/load instructions"


Summary:
Add details on how to save and load quantized models without hitting errors
Address issue #69426
Test Plan:
cd docs
make html

preview docs page, looks good.

![image](https://user-images.githubusercontent.com/2657489/145655866-f1bf7917-6861-412b-9703-676dff63263d.png)


Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33030991](https://our.internmc.facebook.com/intern/diff/D33030991)

[ghstack-poisoned]
